src/ess/amor/geometry.py

```diff
@@ -0,0 +1,64 @@
+import scipp as sc
+
+
+class Detector:
+    'Description of the geometry of the Amor detector'
+    # number of active blades in the detector
+    nBlades = sc.scalar(14)
+    # number of wires per blade
+    nWires = sc.scalar(32)
```

Comment:
The issue is that later on we create the coordinates of the pixels. They are defined something like `xPos = logical_pixel_x_index * dX`. If `logical_pixel_index` has unit `None` that operation fails, because `dX` has unit `mm`.
That's why I made the wire/blade numbering dimensionless.

Is there a canonical way to solve this kind of issues when using `unit=None` quantities?

Reply:
Hmm, would it make sense to use `sc.linspace` instead of multiplication?

Reply:
Not really, that would require some larger refactoring.
I'll leave the numbers as dimensionless for now. If we come up with a better way of doing it we can change it later.